controllers/projectGrading/projectGrading.spec.js

```diff
@@ -177,4 +178,10 @@ describe('judges can [POST, GET, PUT, DELETE] grade a submitted project', () =>
     expect(response15.status).toEqual(200);
     done();
   });
+  test('Check if projectId is a number', () => {
+    expect(projectId).not.toBeNaN();
+  });
+  test('Check if eventId is a number', () => {
+    expect(eventId).not.toBeNaN();
```

Comment:
Can you try something like expect(typeof eventId).toBe('number')

Something that explicitly checks it is a number instead?